box/gen.go

```diff
@@ -90,5 +90,4 @@ func main() {
 	}
 
 	log.Println("Baking resources done... \U0001F680")
-	log.Println("DO NOT COMMIT box/blob.go \U0001F47B")
```

Comment:
@frrist I don't know what this scary-looking line is for. The file is already committed.

Reply:
Its not important, removal here is file. The scary bit at the end is an emoji.